<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectorDialog</class>
 <widget class="QDialog" name="SelectorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Projet TALN - Sélectionner</string>
  </property>
  <property name="locale">
   <locale language="French" country="France"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="selectGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Sélectionner une partie du Quran</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="souratLabel">
        <property name="text">
         <string>Sourate</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QComboBox" name="souratComboBox"/>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="ayatComboBox"/>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="wordComboBox"/>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="ayatCheckBox">
        <property name="text">
         <string>Verset</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="wordCheckBox">
        <property name="text">
         <string>Mot</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="selectedTextEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonsBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonsBox</sender>
   <signal>accepted()</signal>
   <receiver>SelectorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonsBox</sender>
   <signal>rejected()</signal>
   <receiver>SelectorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
